examples/property_estimator/start_client.py

```diff
@@ -8,22 +8,43 @@
 from propertyestimator.utils import get_data_filename, setup_timestamp_logging
 
 
-def submit_calculation_to_server():
+def request_estimate_blocking():
     """Submit calculations to a running server instance"""
+    setup_timestamp_logging()
+
+    # Load in the data set of interest.
+    data_set = ThermoMLDataSet.from_file_list(get_data_filename('properties/single_density.xml'))
+    # Load in the force field to use.
+    force_field = smirnoff.ForceField(get_data_filename('forcefield/smirnoff99Frosst.offxml'))
+
+    # Create the client object.
+    property_estimator = client.PropertyEstimatorClient()
+    # Submit the request to a running server, and wait for the results.
+    result = property_estimator.estimate(data_set, force_field)
 
+    logging.info('The server has returned a response: {}'.format(result))
+
+
+def request_estimate_non_blocking():
+    """Submit calculations to a running server instance"""
     setup_timestamp_logging()
 
+    # Load in the data set of interest.
     data_set = ThermoMLDataSet.from_file_list(get_data_filename('properties/single_density.xml'))
+    # Load in the force field to use.
     force_field = smirnoff.ForceField(get_data_filename('forcefield/smirnoff99Frosst.offxml'))
 
-    property_estimator = client.PropertyEstimator()
-    ticket_ids = property_estimator.submit_computations(data_set, force_field)
+    # Create the client object.
+    property_estimator = client.PropertyEstimatorClient()
+    # Submit the request to a running server.
+    ticket_id = property_estimator.estimate(data_set, force_field)
```

Comment:
I thought we had the idea to move away from returning `ticket_id`s, instead returning a `PropertyEstimate` or `PropertyEstimatResult` result object that either contained the resulting estimate (after a synchronous call) or knew how to query/retrieve the asynchronous estimate (after an async call). Or maybe it could even retrieve the estimate in the background, or when first queried for the result.